addons/website_sale_delivery/models/delivery.py

```diff
@@ -9,4 +9,3 @@ class DeliveryCarrier(models.Model):
     _inherit = ['delivery.carrier', 'website.published.multi.mixin']
 
     website_description = fields.Text(related='product_id.description_sale', string='Description for Online Quotations', readonly=False)
-    website_published = fields.Boolean(default=False)
```

Comment:
Is that ok in stable?
website_published field already in website.published.multi.mixin 2 lines above.